tests/normalisation_patching_test/simple_heap_profiler.wat

```diff
@@ -0,0 +1,182 @@
+;; NOTE expected result is long due too reduction counter logic
+;; Input
+(module
+    (type (;0;) (func))
+    (type (;1;) (func (param i32)))
+    (type (;2;) (func (param i32) (result i32)))
+    (type (;3;) (func (param i32 i32)))
+    (type (;4;) (func (param i32 i32) (result i32)))
+    (type (;5;) (func (param i32 i32 i32)))
+    (func $malloc (type 2) (param i32) (result i32)
+        i32.const 0
+    )
+    (func $aligned_alloc (type 4) (param i32 i32) (result i32)
+        i32.const 0
+    )
+    (func $calloc (type 4) (param i32 i32) (result i32)
+        i32.const 0
+    )
+    (func $realloc (type 4) (param i32 i32) (result i32)
+        i32.const 0
+    )
+    (func $free (type 1) (param i32)
+    )
+)
+
+;; EXPECTED-RESULT:
+(module
+    (type (;0;) (func))
+    (type (;1;) (func (param i32)))
+    (type (;2;) (func (param i32) (result i32)))
+    (type (;3;) (func (param i32 i32)))
+    (type (;4;) (func (param i32 i32) (result i32)))
+    (type (;5;) (func (param i32 i32 i32)))
+    (import "lunatic" "yield_" (func (;0;) (type 0)))
+    (import "heap_profiler" "malloc_profiler" (func (;1;) (type 3)))
+    (import "heap_profiler" "aligned_alloc_profiler" (func (;2;) (type 5)))
+    (import "heap_profiler" "calloc_profiler" (func (;3;) (type 5)))
+    (import "heap_profiler" "realloc_profiler" (func (;4;) (type 5)))
+    (import "heap_profiler" "free_profiler" (func (;5;) (type 1)))
+    (func $malloc_wrap (type 2) (param i32) (result i32)
+        block  ;; Reduction counter logic
```

Comment:
this test is unnecessary long due to reduction logic. We could change the way we test normalisation to shorten some of these (and future) examples